commands/themebuider.js

```diff
@@ -126,10 +126,14 @@ const installThemeBuilder = async version => {
 };
 
 const getDevExtremeVersion = () => {
-    const installedDevExtremePackageJson = path.join(process.cwd(), 'node_modules', 'devextreme', 'package.json');
-    if(fs.existsSync(installedDevExtremePackageJson)) {
-        return require(installedDevExtremePackageJson).version;
+    const lockFileName = path.join(process.cwd(), 'package-lock.json');
```

Comment:
What about yarn.lock?

Reply:
Does it need for app templates? yarn.lock will not be enough in some cases https://supportcenter.devexpress.com/ticket/details/t854678/devextreme-cli-a-theme-is-generated-using-a-global-devextreme-version-instead-of-using

Reply:
We are going to support `yarn` in future